src/Sulu/Component/Webspace/Tests/Unit/Analyzer/Attributes/AdminRequestProcessorTest.php

```diff
@@ -47,12 +47,13 @@ public function testProcess(array $expected = [], $webspaceKey = null, $locale =
         $request->get('locale', $language)->willReturn($locale ?: $language);
         $request->get('language')->willReturn($language);
 
-        $localization = $this->prophesize(Localization::class);
-
         $webspace = $this->prophesize(Webspace::class);
         $webspace->getKey()->willReturn($webspaceKey);
+
+        $localization = null;
         if ($locale || $language) {
```

Comment:
I would move that `||` part out if the if in a separate variable, so that you don't have to do `$locale ?: $language` twice in the block afterwards.